### PR TITLE
Fix network port scan for MacOS

### PR DESCRIPTION
**Pull request by pgrawehr**
Started failing without code changes on our end.

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/iot/pull/2318)

### CODE REVIEW

**Comment by pgrawehr** [comment]
@joperezr Please have a look at this. It fixes the unit test that started to fail on MacOS without any changes on our end. 
Could be a new instance of https://github.com/actions/runner-images/issues/1335 causing #1927 one more.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -150,7 +150,7 @@ stages:
     steps:
     - script: ./build.sh --ci
         --configuration $(BuildConfiguration)
-        --prepareMachine
+        --prepareMachine /maxCpuCount:1
```
Comment (joperezr) [inline azure-pipelines.yml:153]: Can we add a tracking issue and a comment here to make sure we remember to remove this in the future? 

Also can we make this only set in macos? That way we wont be setting this in distros that don't need that (and that are more important like Linux) where we could be unintentionally hiding race condition issues.

**Review by joperezr — APPROVED** [review]
Left a comment but other than that this looks good.

File: azure-pipelines.yml (hunk 1 above)
Comment (joperezr) [reply azure-pipelines.yml:153]: Nevermind, as you pointed out this is only on macos.

**Pull request merged**